db/migrate/20230210115830_add_qualified_teachers_api_status_and_sent_to_qualified_teachers_api_at_to_participant_outcomes.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddQualifiedTeachersApiStatusAndSentToQualifiedTeachersApiAtToParticipantOutcomes < ActiveRecord::Migration[6.1]
+  def change
+    add_column :participant_outcomes, :qualified_teachers_api_request_successful, :boolean, null: false, default: false
```

Comment:
could we remove the `null:false` and default, as we would like to distinguish between the following values:

true: response successful
false: error in response
`nil`: has not been sent to TRA

Reply:
All sorted tks @cwrw 👍